helpers/QueryHelper.php

```diff
@@ -404,6 +437,44 @@ public static function get_all( string $table, array $where, string $order_by, $
 	}
 
 
+	/**
+	 * Get all row from any table with where clause
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param string $table  table name with prefix.
+	 *
+	 * @param string  $column  column name from where we will run the where in clause
+	 * @param string $order_by  order by column name.
+	 * @param string $order_direction  DESC or ASC, default is DESC.
+	 * @param int    $limit default is 1000.
+	 * @param string $output  expected output type, default is object.
+	 *
+	 * @return mixed
+	 */
+	public static function get_all_by_array( string $table, string $column, array $values , string $order_by, string $order_direction = 'DESC', $limit = 1000, string $output = 'OBJECT' ) {
```

Comment:
we already have get_all
`public static function get_all( string $table, array $where, string $order_by, $limit = 1000, string $order = 'DESC', string $output = 'OBJECT' ) `